src/models/Organism.tsx

```diff
@@ -1,20 +1,9 @@
 import { types } from 'mobx-state-tree';
+import { observable } from 'mobx';
 import { v4 as uuid } from 'uuid';
 import { Organelle, IOrganelle, OrganelleType } from './Organelle';
 import { SubstanceType } from './Substance';
 
-export const ModelProperties = types
```

Comment:
This `Type` seems theoretically useful but...it's not used at the moment so I pulled it out